src/node/channels.h

```diff
@@ -417,6 +417,15 @@ namespace ccf
       // the non-authenticated plaintex payload. data contains payload first,
       // then GCM header
 
+      if (status != ESTABLISHED)
```

Comment:
Unrelated but discussed with @wintersteiger: we shouldn't accept messages if the channel isn't ready.